spec/unit/food_truck_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../../app/food_truck'
+require_relative '../../constants'
+
+module FoodTruck
+  RSpec.describe FoodTruck do
+    describe 'FoodTruck loads data' do
```

Comment:
Is that because ScheduleParser only has one It block or because it's never expected to have behavior other than loading data?